hydradx/tests/test_arbitrage_agent_refactor.py

```diff
@@ -110,7 +110,7 @@ def test_calculate_arb_amount_bid(
         sell_ex_max_sell=init_agent.holdings[tkn],
         buy_ex_max_sell=init_agent.holdings[numeraire],
         precision=p,
-        max_iters=100
+        max_iters=110
```

Comment:
This seems really high, was this change really needed to get sufficient accuracy?

Reply:
This came about because I was getting one of those "surprise" test fails (where a test that had been passing suddenly doesn't). Long story short, I forgot to change it back after I figured out what the problem actually was. I'll change it back.